src/libs/tagging.c

```diff
@@ -2936,10 +2936,10 @@ static gboolean _lib_tagging_tag_redo(GtkAccelGroup *accel_group, GObject *accel
     GList *imgs = dt_view_get_images_to_act_on(TRUE, TRUE);
     const gboolean res = dt_tag_attach_string_list(d->last_tag, imgs, TRUE);
     if(res) dt_image_synch_xmps(imgs);
-
+    g_list_free(imgs);
```

Comment:
please note that in  #5405 @AlicVB specifically removed all my calls to `g_list_free` and the `dt_view_get_images_to_act_on` actually returns `darktable.view_manager->act_on.images` so that free isn't needed

Reply:
You may well be right for this one!

Reply:
Guys, that's very dangerous to change the code this way !
All the previous code is based on the fact the caller owns the list and can change it as needed (add grouped image for example). See dt_image_set_location() for example.
And it remains g_list_free(imgs) in various places, even in tagging!

Reply:
I think @AlicVB removed all those that referenced list returned from `dt_view_get_images_to_act_on`. Not many calls were there tbh, and those which needed list ownership were changed to receive list copy.

I'd wait for @AlicVB to review & comment. as it stands now I don't think we have leak nor do I see double frees. sure there might be a lot of `g_list_free` but may not reffer to list returned from images to act on.